lr_face/models.py

```diff
@@ -34,6 +36,28 @@ def __init__(self):
         super().__init__([Input(shape=(100, 100, 3)), Flatten(), Dense(100)])
 
 
+class FaceRecognition(tf.keras.Sequential):
+    """
+    A Face Recognition model that takes RGB images with dimensions as
+    input and outputs random embeddings with dimensionality 128. This keras
+    model won't be used, but the functions will be called directly from the
+    face_recognition library. Resolution (100, 100) is not used.'
+    """
+
+    def __init__(self):
+        super().__init__([Input(shape=(100, 100, 3)), Flatten(), Dense(128)])
+
+    def predict(self, x):
+        embed = None
+        # embed = np.ones(128)
+        try:
+            embed = face_recognition.face_encodings(x)[0]
```

Comment:
I get this error here `RuntimeError: Unsupported image type, must be 8bit gray or RGB image.`